crontab.bak

```diff
@@ -0,0 +1,30 @@
+# Edit this file to introduce tasks to be run by cron.
+# 
+# Each task to run has to be defined through a single line
+# indicating with different fields when the task will be run
+# and what command to run for the task
+# 
+# To define the time you can provide concrete values for
+# minute (m), hour (h), day of month (dom), month (mon),
+# and day of week (dow) or use '*' in these fields (for 'any').
+# 
+# Notice that tasks will be started based on the cron's system
+# daemon's notion of time and timezones.
+# 
+# Output of the crontab jobs (including errors) is sent through
+# email to the user the crontab file belongs to (unless redirected).
+# 
+# For example, you can run a backup of all your user accounts
+# at 5 a.m every week with:
+# 0 5 * * 1 tar -zcf /var/backups/home.tgz /home/
+# 
+# For more information see the manual pages of crontab(5) and cron(8)
+# 
+# m h  dom mon dow   command
+MAILTO=liu472@iu.edu
+SHELL=/bin/bash
+
+#####################################################################
+# Twitter Data Collection
+# grab past month twitter data with new iffy list
+0 8 2 * * /home/data/apps/topfibers/repo/scripts/data_collection/iffy_get_data.sh
```

Comment:
Usually, data loading for day `n` would finish during the afternoon of day `n+1`. Since we also wanted to keep our data in compliance and that most deletions happen within 24 hours, it would be nice to wait until the deletion process completes processing deletion data of day `n+1` in the morning of day `n+2`.

If we plug in the last day of the previous month for `n`, we would get 2<sup>nd</sup> for `n+2`.

Reply:
It seems like this will always be a problem though, right? Any time after we make this call, new deletions could come in that we will have missed for this analysis. Since we are presenting aggregate data and do not show content but simply link to the posts (which will be down as they've been deleted on Twitter), I would prefer to run this as soon as possible.

Am I missing something here?

Reply:
Yeah, I agree. 

Then do you want to run this as soon as the data loading for the last day of the previous month finishes(around 3~4PM) or just 00:01 1<sup>st</sup> day of the month?

Reply:
Yeah, let's do it once we know for sure all of the data we are trying to pull has been loaded to moe. Feel free to buffer in an hour or two to make sure that we know everything is loaded (as I'm assuming that this will take a variable amount of time).